ntc_templates/templates/cisco_xr_show_bgp_vrf_all_neighbors_advertised-routes.textfsm

```diff
@@ -0,0 +1,21 @@
+Value NETWORK (\d+\.\d+\.\d+\.\d+)
+Value PREFIX_LENGTH (\d+)
+Value NEXT_HOP (\d+\.\d+\.\d+\.\d+)
+Value FROM ((\d+\.\d+\.\d+\.\d+)|\S+)
+Value AS_PATH (.*)
+Value Filldown ROUTE_DISTINGUISHER (\d+\:\d+)
+Value Filldown VRF (\S+)
+
+Start
+  ^\S{3}\s+\S{3}\s+\d{1,2}\s+\d+:\d+:\d+
+  ^Network\s+Next\s+Hop\s+From\s+AS\s+Path
+  ^Route\s+Distinguisher\:\s+${ROUTE_DISTINGUISHER}\s+\(default\s+for\s+vrf\s+${VRF}\)\s*$$ -> ROUTES
+  ^Processed\.* -> End
```

Comment:
I anchored the line with AS Path column header at the end.

I also added the various headings should someone not filter for a single VRF.

@jorlandobr
I would add a second raw output file so that we test both a single VRF and multiple VRFs.
I'm going to post in the discussion the mocked up raw output I used for testing.
Thank you!

```suggestion
  ^VRF:\s+\S+:\d+$$
  ^-+$$
  ^%\s+Neighbor\s+not\s+found$$
  ^Network\s+Next\s+Hop\s+From\s+AS\s+Path$$
  ^Route\s+Distinguisher\:\s+${ROUTE_DISTINGUISHER}\s+\(default\s+for\s+vrf\s+${VRF}\)\s*$$ -> ROUTES
  ^Processed\s+
```

Reply:
```
Tue Nov 26 00:00:10.176 BRA

VRF: CITY_GER:7688
-------------------
% Neighbor not found

VRF: MARKET_INT_2:4776
---------------------
% Neighbor not found

VRF: NONAMENEWB:7609
---------------------
Network            Next Hop        From            AS Path
Route Distinguisher: 4230:7609 (default for vrf NONAMENEWB:7609)
0.0.0.0/0          172.23.33.9     10.43.78.102  4230 65000?
10.44.97.0/24      172.23.33.9     10.43.78.102  4230 65000?
10.44.255.184/29   172.23.33.9     10.43.78.102  4230 65000i

Processed 3 prefixes, 3 paths

VRF: VRF_CFTV:7129
----------------------
Network            Next Hop        From            AS Path
Route Distinguisher: 4230:7199 (default for vrf VRF_CFTV:7129)
0.0.0.0/0          172.23.33.9     10.43.78.102  4230 65000?
10.44.97.0/24      172.23.33.9     10.43.78.102  4230 65000?
10.44.255.184/29   172.23.33.9     10.43.78.102  4230 65000i
10.58.0.0/24       172.23.33.9     10.43.78.102  4230 64611i
10.58.8.0/22       172.23.33.9     10.43.78.102  4230 65000i

Processed 5 prefixes, 5 paths

VRF: VRDUF_ACMEUF:9795
----------------------
% Neighbor not found

VRF: VRDUF_VRFCAR:9128
----------------------
% Neighbor not found
```